src/server/game/Guilds/GuildMgr.h

```diff
@@ -47,6 +47,9 @@ class TC_GAME_API GuildMgr
     Guild* GetGuildByName(std::string_view guildName) const;
     std::string GetGuildNameById(ObjectGuid::LowType guildId) const;
 
+    // --- new read-only accessor ---
+    auto const& GetGuildStore() const { return GuildStore; }
```

Comment:
Don't use auto, specify the type explicitly (`GuildContainer`), if neccessary, move the typedef above this definition